README.md

```diff
@@ -11,111 +16,148 @@
+BaritoneAPI.getProvider().getPrimaryBaritone().getCustomGoalProcess().setGoalAndPath(new GoalXZ(10000, 20000));
+```
 
-- [Usage (chat control)](USAGE.md)
+## FAQ
 
-## Stars over time
+*What inspired Baritone?*
 
-[![Stargazers over time](https://starchart.cc/cabaletta/baritone.svg)](https://starchart.cc/cabaletta/baritone)
+Baritone is an updated version of [MineBot](https://github.com/leijurv/MineBot/),
+the original version of the bot for Minecraft 1.8.9, rebuilt for 1.12.2
+through 1.16.5. Baritone focuses on reliability and particularly performance 
+(it's over[30x faster](https://github.com/cabaletta/baritone/pull/180#issuecomment-423822928) than
+MineBot at calculating paths).
 
-# API
+***
 
-The API is heavily documented, you can find the Javadocs for the latest release [here](https://baritone.leijurv.com/).
-Please note that usage of anything located outside of the ``baritone.api`` package is not supported by the API release
-jar.
+*How can I support Baritone?*
 
-Below is an example of basic usage for changing some settings, and then pathing to an X/Z goal.
+You can help support Baritone by contributing or by donating to **1Leijurv3DWTrGAfmmiTphjhXLvQiHg7K2**
 
-```
-BaritoneAPI.getSettings().allowSprint.value = true;
-BaritoneAPI.getSettings().primaryTimeoutMS.value = 2000L;
+***
 
-BaritoneAPI.getProvider().getPrimaryBaritone().getCustomGoalProcess().setGoalAndPath(new GoalXZ(10000, 20000));
-```
+*Can I use Baritone in my utility client?*
+
+Absolutely, as long as your usage is compliant
+with Baritone's [LGPLv3 license](https://github.com/cabaletta/baritone/blob/master/LICENSE).
 
-# FAQ
+More information about the LGPLv3 license can be found [here](https://www.gnu.org/licenses/lgpl-3.0.en.html).
 
-## Can I use Baritone as a library in my custom utility client?
+***
 
-That's what it's for, sure! (As long as usage complies with the LGPL 3.0 License)
+*When will support for Minecraft 1.17.~ be released?*
 
-## How is it so fast?
+Join the [Baritone Discord Server](https://discord.com/invite/s6fRBAUpmr) to be notified of future releases.
 
-Magic. (Hours of [leijurv](https://github.com/leijurv/) enduring excruciating pain)
+***
 
-### Additional Special Thanks To:
+*Is using Baritone cheating?*
+
+No, Baritone is a utility mod intended to automate certain game mechanics.
+
+Do **NOT** use Baritone on public servers which explicitly ban utility clients and other mods which may give
+you an unfair advantage over other players.
+
+***
+
+*How do I use Baritone?*
+
+Read [Usage.md](USAGE.md) or 
+watch [Impact Client's YouTube playlist.](https://www.youtube.com/playlist?list=PLnwnJ1qsS7CoQl9Si-RTluuzCo_4Oulpa)
+
+***
+
+*How does Baritone work and how is it so fast?*
+
+~~Read the source code / documentation~~. 
+
+**Magic** (Over 2 years of [leijurv](https://github.com/leijurv/) enduring excruciating pain)
+
+***
+
+*Why is this project named "Baritone?"*
+
+Read: [Baritone](https://en.wikipedia.org/wiki/Baritone)
```

Comment:
I generally like the idea of giving the Wikipedia link as a part of the explanation, but it feels a bit odd how you embedded it.